config/application.yml

```
@@ -100,6 +100,8 @@ spring:
             uri: ${geonetwork.core.url}
             predicates:
               - Path=/geonetwork/**
+            filters:
+              #- addSimpleGn4SecurityHeader=gn5.to.gn4.trusted.json.auth
```

@davidblasby we should enable simple mode by default no?

```suggestion
              - addSimpleGn4SecurityHeader=gn5.to.gn4.trusted.json.auth
```